EIPS/eip-7768.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: 7768
+title: No-Ether transactions with free-for-all tips
+description: Externally-owned account having no Ether can send transactions and pay tips using a new "free-for-all" bucket
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-7768-no-ether-transactions-with-free-for-all-tips/21108
+status: Draft
+type: Meta
+created: 2024-09-14
+---
+
+## Abstract
+
+A technique is introduced where an externally-owned account having no Ether can send transactions and pay tips using a new "free-for-all" bucket and using their own `origin.tx`. This requires no client changes and is compatible with existing ecosystem parts.
```

Comment:
You should provide a high-level (but still technical) overview here of how this works. For example, what's a bucket? How does the block producer get paid?

Reply:
Thank you, updated at https://github.com/fulldecent/EIPs/blob/a2dbd017621aa223e6be669bf3ccae8475938cbb/EIPS/eip-7768.md